client/src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {Navigation} from 'swiper'
+import {Typography, styled} from "@mui/material";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import {CardItem} from '../index'
+
+
+import 'swiper/swiper.min.css'
+
+
+const cards = [
+    {
+        name: "Kyiv city walk",
+        region: "Kyiv region",
+        id: 1,
+        url: ""
+    },
+    {
+        name: "Lviv city walk",
+        region: "Lviv region",
+        id: 2,
+    },
+    {
+        name: "Donetsk city walk",
+        region: "Donetsk region",
+        id: 3,
+    },
+    {
+        name: "Odessa city walk",
+        region: "Odessa region",
+        id: 4,
+    },
+    {
+        name: "Dnepr city walk",
+        region: "Dnepr region",
+        id: 5,
+    },
+    {
+        name: "Chernivtsi city walk",
+        region: "Chernivtsi region",
+        id: 6,
+    },
+    {
+        name: "Ternopil city walk",
+        region: "Ternopil region",
+        id: 7,
+    },
+    {
+        name: "Lutsk city walk",
+        region: "Lutsk region",
+        id: 8,
+    },
+]
+
+const CardContainerWrapper = styled('div')({
+    maxWidth: "1150px",
+    margin: "0 auto",
+    padding: "0 20px",
+})
+
+const CardContainerHeaderContainer = styled('div')({
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+})
+
+const CardContainerHeader = styled(Typography)(({theme}) => ({
+    fontSize: "24px",
+    color: theme.palette.primary.main,
+    marginBottom: "0",
+}))
+
+
+const CardContainerAllTours = styled(Typography)({
+    fontSize: "12px",
+    color: "#282828",
+})
+
+const CardContainerSwiperContainer = styled('div')({
+    position: "relative",
+    margin: "70px auto 0 auto",
+
+    '@media (min-width: 300px)': {
+        width: "265px",
+    },
+
+    '@media (min-width: 600px)': {
+        width: "560px",
+    },
+
+    '@media (min-width: 900px)': {
+        width: "860px",
+    },
+
+    '@media (min-width: 1200px)': {
+        width: "1150px",
+    },
+})
+
+const CardContainerBtnPrev = styled('div')(({theme}) => ({
+    backgroundColor: "white",
+    height: "60px",
+    width: "60px",
+    borderRadius: "50px",
+    position: "absolute",
+    cursor: "pointer",
+    zIndex: "2",
+    top: "50%",
+    marginTop: "-30px",
+    marginLeft: "-30px",
+
+    "& .MuiSvgIcon-root": {
+        position: "absolute",
+        top: "50%",
+        left: "50%",
+        marginLeft: "-12px",
+        marginTop: "-10px",
+        rotate: "180deg",
+        color: theme.palette.primary.main,
+    }
+}))
+
+const CardContainerBtnNext = styled('div')(({theme}) => ({
+    backgroundColor: "white",
```

Comment:
please, change to `backgroundColor: theme.palette.primary.contrast text,`

Reply:
done